front/app/components/EventPreviews/EventPreviewCard.tsx

```diff
@@ -76,13 +76,13 @@ const EventPreviewCard = ({ event }: EventPreviewCardProps) => {
         theme.isRtl ? `4px solid ${theme.colors.tenantPrimary}` : undefined
       }
       w={isTablet ? (isMobile ? '100%' : '300px') : '340px'}
-      role="button"
```

Comment:
Good that you removed this as a first step, but these are not reachable by keyboard because li's need to have their tabindex set to 0, as we do with the regular event cards.